File: desktop/src/main/scala/gdx/scala/demo/desktop/DesktopLauncher.java
package gdx.scala.demo.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import gdx.scala.demo.GdxScalaDemoGame;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		config.title = "Pong";
		config.width = 800;
		config.height = 480;
		new LwjglApplication(new GdxScalaDemoGame(), config);
	}
}

File: desktop/src/main/scala/gdx/scala/demo/desktop/DesktopLauncher.scala
package gdx.scala.demo.desktop

import com.badlogic.gdx.backends.lwjgl.{LwjglApplication, LwjglApplicationConfiguration}
import gdx.scala.demo.PongGame

object DesktopLauncher extends App {
  val config = new LwjglApplicationConfiguration()
  config.title = "Pong"
  config.width = 800
  config.height = 480
  val game = new PongGame(config.width, config.height)
  val app = new LwjglApplication(game, config)
}
